flash-messages/.gitignore

```diff
@@ -0,0 +1,31 @@
+.DS_Store
+.env
+*.db
+
+*.gem
+*.rbc
+/.config
+/coverage/
+/InstalledFiles
+/pkg/
+/spec/reports/
+/test/tmp/
+/test/version_tmp/
+/tmp/
+
+## Specific to RubyMotion:
+.dat*
+.repl_history
+build/
+
+## Documentation cache and generated files:
+/.yardoc/
+/_yardoc/
+/doc/
+/rdoc/
+
+## Environment normalisation:
+/.bundle/
+/lib/bundler/man/
+
+.rvmrc
```

Comment:
Big old gitignore! Is it possible we don't need all this for this specific example?